docs/ldml/tr35-keyboards.md

```diff
@@ -1478,7 +1500,7 @@ _Attribute:_ `id` (required)
 
 _Attribute:_ `value` (required)
 
-> Strings may contain whitespaces. However, for clarity, it is recommended to escape spacing marks, even in strings.
+> Strings may contain whitespaces. However, for clarity, it is recommended to escape non-spacing marks, even in strings.
```

Comment:
I think this was original supposed to refer to whitespace characters rather than "spacing marks" (which is kinda weird).

```suggestion
> Strings may contain whitespaces. However, for clarity, it is recommended to escape whitespace characters, even in strings.
```